### PR TITLE
blog: Introduce the community forum

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
For sharing a blog article is nice, so this is a post to that regard.

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2023/06/introducing-the-flowforge-community-forum.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Introducing the FlowForge Community Forum
+subtitle: A Community Forum for support, inspiration, and knowledge sharing
+description: 
+date: 2023-06-09 12:00:00
+authors: ["zeger-jan-van-de-weg"]
+tags:
+  - news
+  - flowforge
+image: # /images/blog/tile-needed.jpg
```
Comment (ZJvandeWeg) [inline src/blog/2023/06/introducing-the-flowforge-community-forum.md:10]: @iskerrett I like the generated tile, and don't want to wait on a "nice" tile. I'd be fine with merge with this tile.

File: src/blog/2023/06/introducing-the-flowforge-community-forum.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Introducing the FlowForge Community Forum
+subtitle: A Community Forum for support, inspiration, and knowledge sharing
+description: 
+date: 2023-06-09 12:00:00
```
Comment (ZJvandeWeg) [inline src/blog/2023/06/introducing-the-flowforge-community-forum.md:5]: I think the target date for tomorrow should be better than today?

Comment (iskerrett) [reply src/blog/2023/06/introducing-the-flowforge-community-forum.md:5]: Changing to days date.

Comment (iskerrett) [inline src/blog/2023/06/introducing-the-flowforge-community-forum.md:5]:
```suggestion
date: 2023-06-0912 12:00:00
```

Comment (iskerrett) [inline src/blog/2023/06/introducing-the-flowforge-community-forum.md:5]:
```suggestion
date: 2023-06-12 12:00:00
```

**Review by iskerrett — APPROVED** [review]

**Pull request merged**